.drone.yml

```
@@ -0,0 +1,313 @@
+---
+kind: pipeline
+name: x86
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: dockerfiles
+  image: alpine
+  commands:
+  - ./dockerfiles.sh
+
+- name: v3.6
+  image: plugins/docker
+  settings:
+    dockerfile: Dockerfiles/v3.6/x86/Dockerfile
+    password:
+      from_secret: docker_pass
+    repo: clandmeter/docker-abuild
+    tags: 3.6-x86
+    username:
+      from_secret: docker_user
```

```suggestion
      from_secret: docker_user
  depends_on:
    - dockerfiles
```

Also, add a similar entry for every release step. We should be able to build release images in parallel once the `dockerfiles` are generated.